shared/utils/rank.js

```diff
@@ -0,0 +1,2 @@
+// @flow
+export type Rank = "Admin" | "Viewer" | "Member";
```

Comment:
Can move this inside the `types.js` file now

Reply:
It is also being used on the front end. Should I create `shared/types.js`? otherwise we will have to keep it in `app/types/index.js` and `server/types.js`

Reply:
>  Should I create `shared/types.js? `

Yes let's do that, I'm amazed it hasn't been needed yet.